### PR TITLE
Update readme with Video and details

### PR DESCRIPTION
**Pull request by patched-admin**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -120,11 +117,31 @@ Each patchflow comes with an optimized default prompt template. But you can spec
...
 
-To create a new patchflow, follow [these instructions](patchwork/patchflows/README.md).
+Contributions for new patchflows and steps, or even to the core framework are welcome. Please look at open issues for details.
 
-To create a new step, follow [these instructions](patchwork/steps/README.md).
+- To create a new patchflow, follow [these instructions](patchwork/patchflows/README.md).
+- To create a new step, follow [these instructions](patchwork/steps/README.md).
 
-We also provide chat assistants to help you create new steps and patchflows easily.
+We also provide chat assitants to help you create new steps and patchflows easily. Fair warning: they suffer from the same limitations as their underlying model.
```
Comment (codelion) [inline README.md:125]: Can we fix the typo “assitants” ~~> “assistants”

**Comment by patched-admin** [comment]
The code changes include instructions for installing PatchWork using Pip and Poetry, as well as a new Patchwork logo and various formatting improvements in the README.md file. The updated README now includes links to a demo and documentation, additional shields for Build Status, Downloads, and Discord, expanded sections with detailed explanations of key components, installation methods, Patchwork CLI usage, and contribution guidelines, as well as upcoming features in the roadmap and licensing details.

------

<div markdown="1">

* File changed: [INSTALL.md](https://github.com/patched-codes/patchwork/pull/90/files#diff-09b140a43ebfdd8dbec31ce72cafffd15164d2860fd390692a030bcb932b54a0)
  [The diff provides instructions for installing PatchWork, a tool available on PyPI, using both Pip and Poetry. It outlines the steps for installing Poetry if not previously installed, cloning the PatchWork repository, navigating to the project directory, activating a shell using virtual environment, and installing the dependencies using Poetry.](https://github.com/patched-codes/patchwork/pull/90/files#diff-09b140a43ebfdd8dbec31ce72cafffd15164d2860fd390692a030bcb932b54a0)

</div>

<div markdown="1">

* File changed: [README.md](https://github.com/patched-codes/patchwork/pull/90/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5)
  [The README.md file was updated to include a new Patchwork logo, various aligning and formatting improvements, additional shields for Build Status, Downloads, and Discord, as well as links to a demo and documentation. The sections were expanded to include more detailed explanations of key components, the Quickstart guide, installation methods, Patchwork CLI usage, and contribution guidelines, with a mention of upcoming features in the roadmap and licensing details.](https://github.com/patched-codes/patchwork/pull/90/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5)

</div>

**Comment by patched-admin** [comment]
The code changes include instructions for installing PatchWork using Pip and Poetry, as well as updates to the README.md file with changes to the alignment and size of the Patchwork logo, additional badges, demo and documentation links, and a modified structure with sections on Key Components, Quickstart, Installation, PatchWork CLI instructions, Prompt Templates, Contributing guidelines, Roadmap, and License information. Detailed information on various components, creating new patchflows and steps, and a roadmap for Short Term and Long Term goals with licensing information under AGPL-3.0 and Apache-2.0 terms are also included.

------

<div markdown="1">

* File changed: [INSTALL.md](https://github.com/patched-codes/patchwork/pull/90/files#diff-09b140a43ebfdd8dbec31ce72cafffd15164d2860fd390692a030bcb932b54a0)
  [The diff provides instructions for installing PatchWork using Pip and Poetry. It includes commands for installing PatchWork using pip and a step-by-step guide for installing it using Poetry, including cloning the repository, navigating to the project directory, activating a virtual environment, and installing dependencies using Poetry.](https://github.com/patched-codes/patchwork/pull/90/files#diff-09b140a43ebfdd8dbec31ce72cafffd15164d2860fd390692a030bcb932b54a0)

</div>

<div markdown="1">

* File changed: [README.md](https://github.com/patched-codes/patchwork/pull/90/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5)
  [The README.md file has been updated with changes to the alignment and size of the Patchwork logo, additional badges, links to demo and documentation, and a modified structure with sections on Key Components, Quickstart, Installation, PatchWork CLI instructions, Prompt Templates, Contributing guidelines, a Roadmap, and License information. Moreover, there are added details on various components of PatchWork and instructions on creating new patchflows and steps with links to relevant README files. Additionally, a detailed roadmap for Short Term and Long Term goals has been included along with licensing information regarding AGPL-3.0 and Apache-2.0 terms for the project and its template repository.](https://github.com/patched-codes/patchwork/pull/90/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5)

</div>

**Comment by patched-admin** [comment]
The changes in the INSTALL.md file provide detailed instructions for installing PatchWork using Pip and Poetry, including installing dependencies and activating a virtual environment. In the README, updates include changing the logo image, adding aligning divs, shields for build status and downloads, links for Demo and Docs, revising content and adding references to sections for Key Components, Quickstart, PatchWork CLI, and Contributing, as well as adding a new section on Roadmap and License details.

------

<div markdown="1">

* File changed: [INSTALL.md](https://github.com/patched-codes/patchwork/pull/90/files#diff-09b140a43ebfdd8dbec31ce72cafffd15164d2860fd390692a030bcb932b54a0)
  [The diff in the INSTALL.md file provides instructions for installing PatchWork using Pip and Poetry, including commands for installing dependencies and activating a virtual environment.](https://github.com/patched-codes/patchwork/pull/90/files#diff-09b140a43ebfdd8dbec31ce72cafffd15164d2860fd390692a030bcb932b54a0)

</div>

<div markdown="1">

* File changed: [README.md](https://github.com/patched-codes/patchwork/pull/90/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5)
  [The README was updated to change the logo image, add new aligning divs, include shields for build status and downloads, and provide links for Demo and Docs. Additionally, the sections for Key Components, Quickstart, PatchWork CLI, and Contributing were modified to revise content and add references. Finally, a new section on Roadmap and License details were included in the README.](https://github.com/patched-codes/patchwork/pull/90/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5)

</div>

**Comment by patched-admin** [comment]
The code changes in the INSTALL.md file provide updated instructions on how to install PatchWork either through Pip from PyPI or using Poetry, which involves installing Poetry, cloning the PatchWork repository, navigating to the project directory, activating a shell with a virtual environment, and installing dependencies with Poetry. The diff in the README.md file includes updates to the PatchWork logo, alignment and styling improvements, additional badges for build status and downloads, demo and documentation links, details on Prompt Templates, revised installation instructions for Poetry, a Patchwork CLI Quickstart link, information on CLI commands for scenarios like AutoFix, contributions guidance, and details on the project's roadmap and licensing.

------

<div markdown="1">

* File changed: [INSTALL.md](https://github.com/patched-codes/patchwork/pull/90/files#diff-09b140a43ebfdd8dbec31ce72cafffd15164d2860fd390692a030bcb932b54a0)
  [The diff in the INSTALL.md file provides instructions on how to install PatchWork either using Pip from PyPI with a simple command or using Poetry by first installing Poetry, cloning the PatchWork repository, navigating to the project directory, activating a shell using a virtual environment, and then installing the dependencies using Poetry.](https://github.com/patched-codes/patchwork/pull/90/files#diff-09b140a43ebfdd8dbec31ce72cafffd15164d2860fd390692a030bcb932b54a0)

</div>

<div markdown="1">

* File changed: [README.md](https://github.com/patched-codes/patchwork/pull/90/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5)
  [The diff in the README.md file includes changes such as updating the PatchWork logo, adding new alignment and styling to the content, including additional badges for build status and downloads, adding links to demo and documentation, introducing key components like Prompt Templates, updating installation instructions for Poetry, adding a Patchwork CLI Quickstart link, providing details on PatchWork CLI commands for different scenarios like AutoFix, updating contributions instructions, and adding information on the project's roadmap and licensing.](https://github.com/patched-codes/patchwork/pull/90/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5)

</div>

**Review by codelion — APPROVED** [review]

**Comment by patched-admin** [comment]
The INSTALL.md file now includes instructions for installing PatchWork using both Pip and Poetry, providing users with the flexibility to choose their preferred installation method. The README.md has been revamped with new alignments, styling, and additional information, including a detailed section on PatchWork components and features, updated installation and usage instructions using Poetry, and new sections on quick start, CLI, prompt templates, contributions, and the project roadmap. The roadmap outlines goals for enhancing PatchWork in the short, mid, and long term, including stabilizing existing patchflows, improving documentation and user experience, expanding the library, integrating with CI/CD tools, developing AI-powered patchflow generation, supporting multiple repositories, and implementing enterprise-ready features. Updated license information is also provided.

------

<div markdown="1">

* File changed: [INSTALL.md](https://github.com/patched-codes/patchwork/pull/90/files#diff-09b140a43ebfdd8dbec31ce72cafffd15164d2860fd390692a030bcb932b54a0)
  [The diff in the INSTALL.md file provides instructions for installing PatchWork using both Pip and Poetry. Users can choose to install PatchWork via Pip by using the command `pip install patchwork-cli --upgrade`, or opt for Poetry by following a series of steps including cloning the PatchWork repository and using Poetry for dependency management.](https://github.com/patched-codes/patchwork/pull/90/files#diff-09b140a43ebfdd8dbec31ce72cafffd15164d2860fd390692a030bcb932b54a0)

</div>

<div markdown="1">

* File changed: [README.md](https://github.com/patched-codes/patchwork/pull/90/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5)
  [The README.md file for the PatchWork project has been updated with new alignment and styling for images and shields, additional links, and a more detailed section on the PatchWork components and key features. Additionally, the instructions for installation and usage of PatchWork using Poetry have been altered, and new sections about quick start, PatchWork CLI, prompt templates, contributing guidelines, and the project roadmap have been added. The license information has also been updated.](https://github.com/patched-codes/patchwork/pull/90/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5)

</div>

<div markdown="1">

* File changed: [Roadmap.md](https://github.com/patched-codes/patchwork/pull/90/files#diff-184cd41c0fa500ac6ef7894268d24a33f49bc70fa5800bfae0634f809fed713b)
  [The roadmap outlines short-term, mid-term, and long-term goals for enhancing patchwork, including stabilizing existing patchflows, improving documentation and user experience, expanding the patchflow library, integrating with more CI/CD tools, developing AI-powered patchflow generation, supporting multiple repositories, and introducing enterprise-ready features.](https://github.com/patched-codes/patchwork/pull/90/files#diff-184cd41c0fa500ac6ef7894268d24a33f49bc70fa5800bfae0634f809fed713b)

</div>

**Pull request merged**